### ISSUE TITLE
Explore: option to keep newlines

### ISSUE THREAD

**Issue by cdalexndr** [grafana/grafana#25640]
**What would you like to be added**:
Option to display the field value with newlines.

**Why is this needed**:
A java "stacktrace" message field is displayed as a continous line
`at package.class.metod at package.class.metod2`
This display is hard to follow when there are many stack frames and must be manually edited in a text editor to add newlines.

Semantically there are multiple stack frames, one on each line:
```
at package.class.metod
at package.class.metod2
```
This way the stactrace is clear in the view.

Note that the html text seems to contain newline characters. 
With Firefox, right click -> 'Inspect element', right click html element -> 'Edit as Html', then copy paste in text editor shows the text with newlines.

Related: https://github.com/grafana/grafana/issues/17584

**Comment by sakjur** [comment]
Is this about logs in explore, the logs panel, or the table panel?

**Comment by cdalexndr** [comment]
Logs in Explore. Clicking a log entry shows `Parsed Fields`. I have a field named `stacktrace` with this behaviour.

**Comment by Elfo404** [comment]
Hello @cdalexndr , Just to confirm, are you using the "wrap lines" option when dsiplaying logs? this should have been fixed here https://github.com/grafana/grafana/pull/22057, can you confirm this issue is still relevant?

**Comment by cdalexndr** [comment]
Issue still exists in **Grafana v7.2.2 (ad9d408ac2)**.

Wrap lines option doesn't fix this because there are **no newlines** in the field value (they are removed).
What I see with wrap lines off are 3 lines, with it on 8 lines.
Note that this message should have 18 lines as in a text editor.

**Comment by ivanahuckova** [comment]
Hello @cdalexndr we have already similar issue created here #25640. 

/duplicate of https://github.com/grafana/grafana/issues/25640

**Comment by ivanahuckova** [comment]
@cdalexndr Could you please share the log lines (at least 1 line with newline character) from the network tab - directly from response tab. I just want to see newline character and if it is correctly escaped (as it is not visible in log lines). 
![image](https://user-images.githubusercontent.com/30407135/98654389-8b7b7300-233e-11eb-80b5-b3c202dba5cb.png). Also I think that this issue is very similar to https://github.com/grafana/grafana/issues/27964. So I am going to close this in favour of the other one. Please could you share all of this there, so we have all information needed in 1 place. Thank you very much!

**Comment by cdalexndr** [comment]
@ivanahuckova #27964 is not similar to this issue.

Issue #27964 is related to loki log message, but with `Grafana v7.2.2 (ad9d408ac2)` elasticsearch log messages are formatted correctly with newlines.
My issue is related to **parsed field values**, when you expand the log message.

**Comment by cdalexndr** [comment]
Example stacktrace without newlines.
![image](https://user-images.githubusercontent.com/22418250/100516313-af86e300-318b-11eb-979b-6512a19b8f1a.png)

Text from firefox debug tools Inspector:
```
<td class="css-hh1wya-wordBreakAll">	at pse.shop.component.template.ShopTemplateService.getByShop(ShopTemplateService.java:29) ~[classes!/:na]
	at pse.web.admin.template.shop.TemplateBuilderController.isDirty(TemplateBuilderController.java:152) ~[classes!/:na]
	at pse.web.ShallowEtagHeaderFilter.doFilterInternal(ShallowEtagHeaderFilter.java:72) ~[classes!/:na]
	at pse.web.config.HtmlCompressorFilter.doFilter(HtmlCompressorFilter.java:36) ~[classes!/:na]
</td>
```

From json response:
`"stacktrace": "\tat pse.shop.component.template.ShopTemplateService.getByShop(ShopTemplateService.java:29) ~[classes!/:na]\n\tat pse.web.admin.template.shop.TemplateBuilderController.isDirty(TemplateBuilderController.java:152) ~[classes!/:na]\n\tat pse.web.ShallowEtagHeaderFilter.doFilterInternal(ShallowEtagHeaderFilter.java:72) ~[classes!/:na]\n\tat pse.web.config.HtmlCompressorFilter.doFilter(HtmlCompressorFilter.java:36) ~[classes!/:na]\n",`

**Comment by hexwit** [comment]
I have faced the same problem, that stack trace displayed without new lines. 
How '\n' should be escaped to have it displayed correctly, with carriage return?
